app/client/src/plugins/vuetify.js

```diff
@@ -1,8 +1,4 @@
-if (process.env.VUE_APP_FONTAWESOME_NPM_AUTH_TOKEN) {
-  require("@fortawesome/fontawesome-pro/css/all.css");
-} else {
-  require("@fortawesome/fontawesome-free/css/all.css");
-}
```

Comment:
Maybe we can revert this to how it was using font-awesome pro.

Reply:
Yeah sure